### PR TITLE
fix warning so the user has a clear next step

### PR DESCRIPTION
**Pull request by nohalon**
tell the user exactly how to install `lightning-api-access` directly from the warning

cc @borda

### CODE REVIEW

File: src/lightning/app/components/serve/auto_scaler.py [hunk 1]
```diff
@@ -453,7 +453,7 @@ def _get_endpoint_info_page(self) -> Optional["APIAccessFrontend"]:  # noqa: F82
         try:
             from lightning_api_access import APIAccessFrontend
         except ModuleNotFoundError:
-            logger.warn("APIAccessFrontend not found. Please install lightning-api-access to enable the UI")
+            logger.warn("APIAccessFrontend not found. To enable the UI run: pip install lightning-api-access")
```
Comment (tchaton) [inline src/lightning/app/components/serve/auto_scaler.py:456]:
```suggestion
            logger.warn("You are missing some dependencies. Run `pip install lightning-api-access` to resolve the UI.")
```

**Review by tchaton — APPROVED** [review]
LGTM !

File: src/lightning/app/components/serve/python_server.py [hunk 2]
```diff
@@ -293,7 +293,7 @@ def configure_layout(self) -> Optional["Frontend"]:
         try:
             from lightning_api_access import APIAccessFrontend
         except ModuleNotFoundError:
-            logger.warn("APIAccessFrontend not found. Please install lightning-api-access to enable the UI")
+            logger.warn("APIAccessFrontend not found. To enable the UI run: pip install lightning-api-access")
```
Comment (awaelchli) [inline src/lightning/app/components/serve/python_server.py:296]:
```suggestion
            logger.warn("APIAccessFrontend not found. To enable the UI, run: pip install lightning-api-access")
```
Not sure, but I think the comma is missing.

**Review by awaelchli — APPROVED** [review]

File: src/lightning/app/components/serve/auto_scaler.py (hunk 1 above)
Comment (nohalon) [inline src/lightning/app/components/serve/auto_scaler.py:456]:
```suggestion
            logger.warn("Some dependencies to run the UI are missing. To resolve, run `pip install lightning-api-access`")
```

File: src/lightning/app/components/serve/python_server.py (hunk 2 above)
Comment (nohalon) [inline src/lightning/app/components/serve/python_server.py:296]:
```suggestion
            logger.warn("Some dependencies to run the UI are missing. To resolve, run `pip install lightning-api-access`")
```

**Pull request merged**